tuxemon/event/conditions/current_state.py

```diff
@@ -37,4 +41,6 @@ def test(self, session: Session, condition: MapCondition) -> bool:
         """
         current_state = session.client.current_state
         assert current_state
-        return current_state.name == condition.parameters[0]
+        states = condition.parameters[0].split(":")
+        state = True if current_state.name in states else False
```

Comment:
If you want to return a boolean value, you can just directly return the result of the condition.
Meaning this can be simplified to:
```suggestion
        state = current_state.name in states
```

Reply:
... and I see the line below is just about returning this variable, so you can even inline the statements and just have:

```suggestion
        return current_state.name in states
```